experiments/optimal-core-allocation/README.md

```diff
@@ -98,96 +97,88 @@ where:
+- green - no violation,
+- yellow - tail latency between 101% and 150% of SLO, 
+- red - tail latency above 150% of SLO, 
+- gray - memcached was incapable of handling requested amount of QPS.
 
 There are two dimensions:
 - load (x axis) - fraction of peak load expressed in QPS (Queries Per Second)
 - worker threads (y axis) - number of memcached worker threads (check memcached -t option for details).
 
-Results below were run with 10 load points with peak load set to 1.5 million of QPS with SLO set to 500 us.
+The experiment has been run with 10 load points and peak load set to 1.5 million QPS, and SLO set to 500 us.
 
-#### Table 1. normal run default latency SLO = 500us
+### Table 1. normal run default latency SLO = 500us
 
 ![table 1](images/result-local.PNG)
 
-##### Interpreting results
+#### Interpreting results
 
-You can observe, that each additional memcached worker, up to 10 on 16 processor machine, adds service capacity in terms of throughput.
-With strict requirements about latency, you need to take into consider latency SLO violation and remove red and yellow cells from acceptable configurations.
-So the optimal configuration to can handle **900k QPS** still guaranteeing response times (tail latency < 500us) and which uses as little resources as possible is **11 threads**.
-Increasing number of logical threads above **11 threads**, doesn't improve performance and surprisingly (because of hyper threading) can cause service level degradation.
+You can observe, that each additional memcached worker, up to 10 on 16 CPU machine, adds capacity in terms of throughput.
+Taking SLO into consideration this very configuration can handle  **900.000 QPS** while meeting tail latency requirements (equls or less than 500us). You will be able to utilize **11 threads** on the node.
+Increasing number of logical threads further, doesn't improve performance and surprisingly (because of hyperthreading) can cause latency degradation.
 
-#### Table 2. normal run with SLO = 3000 us
+### Table 2. normal run with SLO = 3000 us
 
 ![table 2](images/result-local-slo3000.PNG)
 
-##### Interpreting results
+#### Interpreting results
 
-After increasing the accepted tail latency, i.e. SLO, to 3ms there are additional acceptable configurations we can choose from.
-We can assign less threads which are capable to handle even more load, while still guaranteeing that our new relaxed SLO is met.
+After increasing the accepted tail latency to 3ms there we can choose from broader range of configurations.
+Less threads are necessary to handle requests, while relaxed target SLO is met.
 
-In this case optimal configuration can use **6, 7 or 8 threads**, depending of desired machine utilization, which should be capable of handling more than 1m QPS and with response below 3ms.
+In this case optimal configuration will utilize **7 threads** and will be capable of handling more than 1.000.000 QPS with tail latency below 3ms.
 
-##### Discussion
+#### Discussion
```

Comment:
@ppalucki it would be awsome if we could draw a graph comparing two experiments here :)